src/main/java/dook/commands/TaskType.java

```diff
@@ -0,0 +1,5 @@
+package dook.commands;
+
+public enum TaskType {
```

Comment:
Any reason why you put this in the commands package as opposed to the tasks package?